### PR TITLE
Send component's node id

### PR DESCRIPTION
**Pull request by mlohvynenko**

### CODE REVIEW

File: umcontroller/umcontroller.go [hunk 1]
```diff
@@ -1135,7 +1140,7 @@ func (umCtrl *Controller) processUpdateUpdateStatus(ctx context.Context, e *fsm.
 		}
 	}
 
-	umCtrl.updateCurrentComponentsStatus(status.componsStatus)
+	umCtrl.updateCurrentComponentsStatus(status.componsStatus, umID)
```
Comment (al1img) [inline umcontroller/umcontroller.go:1143]: Using `umID` looks confusing now. We should rename it to `nodeID` in all the cases. Could you do it in separate commit this PR?

File: umcontroller/umcontroller.go
Comment (al1img) [inline umcontroller/umcontroller.go:1]: `umIDfound` need to be renamed

**Review by mykola-kobets-epam — APPROVED** [review]
